### PR TITLE
Add contributing guidelines for apps/meeting demo

### PR DESCRIPTION
**Pull request by devalevenkatesh**
**Issue #:**
- Provide guidance on how to contribute to the meeting demo specifically.

**Description of changes:**
- Add "CONTRIBUTING" guidelines specific to apps/meeting demo.
- This PR must go in the "main" branch as it is adding contributing guidelines. Rest all will go to "meeting-dev" branch as per these newly added guidelines.

**Testing**

1. How did you test these changes?
Documentation update, but tested the steps mentioned in the documentation.

2. Can these changes be tested using one of the demo application? If yes, which demo application can be used to test it?
NA.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apps/meeting/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+Please read through this document before submitting any issues or pull requests to ensure we have all the necessary
+information to effectively respond to your bug report or contribution.
+
+
+## Reporting Bugs/Feature Requests
+
+We welcome you to use the GitHub issue tracker to report bugs or suggest features.
+
+When filing an issue, please check existing open, or recently closed, issues to make sure somebody else hasn't already
```
Comment (jing-chen1) [inline apps/meeting/CONTRIBUTING.md:14]: [nit] would it be better to remove the commas?
`...please check existing open or recently closed issues to make...`

File: apps/meeting/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+* A reproducible test case or series of steps
+* The version of our code being used
+* Any modifications you've made relevant to the bug
+* Anything unusual about your environment or deployment
+
+
+## Contributing via Pull Requests
+Contributions via pull requests are much appreciated. Before sending us a pull request, please ensure that:
+
+To send us a pull request, please:
```
Comment (jing-chen1) [inline apps/meeting/CONTRIBUTING.md:26]: Would it be better to remove this sentence?

File: apps/meeting/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+1. Fork the repository and make sure you are working against the latest source on the *meeting-dev* branch. We use *meeting-dev* branch for `apps/meeting` app to contribute all PRs. This app is used to test the changes releasing into the next [amazon-chime-sdk-component-library-react](https://github.com/aws/amazon-chime-sdk-component-library-react) release.
+2. Run the meeting demo locally first. The *meeting-dev* apps/meeting demo changes are unstable and contributors will need to create and install the `amazon-chime-sdk-component-library-react` tarball from its *master* branch to start the demo.
+
+    1. In a different directory, create `amazon-chime-sdk-component-library-react` tarball.
+    ```
+    git clone https://github.com/aws/amazon-chime-sdk-component-library-react &&
+    cd amazon-chime-sdk-component-library-react &&
```
Comment (jing-chen1) [inline apps/meeting/CONTRIBUTING.md:33]: [for learning] we don't need to `npm run build` first?

File: apps/meeting/CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+
+
+## Code of Conduct
+This project has adopted the [Amazon Open Source Code of Conduct](https://aws.github.io/code-of-conduct).
+For more information see the [Code of Conduct FAQ](https://aws.github.io/code-of-conduct-faq) or contact
+opensource-codeofconduct@amazon.com with any additional questions or comments.
+
+
+## Security issue notifications
+If you discover a potential security issue in this project we ask that you notify AWS/Amazon Security via our [vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/). Please do **not** create a public github issue.
```
Comment (jing-chen1) [inline apps/meeting/CONTRIBUTING.md:77]: Would it be better to change the  `...we ask that you notify...` to `...we ask you to notify...`?

**Review by jing-chen1 — COMMENTED** [review]
Thanks for creating the instruction for external customers! It has lots of details and is comprehensive. Just left few comments

File: apps/meeting/CONTRIBUTING.md (hunk 2 above)
Comment (devalevenkatesh) [reply apps/meeting/CONTRIBUTING.md]: Cool, removed thanks!

File: apps/meeting/CONTRIBUTING.md (hunk 1 above)
Comment (devalevenkatesh) [reply apps/meeting/CONTRIBUTING.md]: Removed from this and the main contributing guide as well. I had just copied over from it 😅 .

File: apps/meeting/CONTRIBUTING.md (hunk 3 above)
Comment (devalevenkatesh) [reply apps/meeting/CONTRIBUTING.md:32]: That is correct, I missed adding the install and build step, nice catch!

File: apps/meeting/CONTRIBUTING.md (hunk 4 above)
Comment (devalevenkatesh) [reply apps/meeting/CONTRIBUTING.md]: Updated to : ", please notify". Thanks!

**Review by jing-chen1 — APPROVED** [review]

**Pull request merged**